miniwallet/.env.example

```diff
@@ -12,13 +12,106 @@ ROPSTEN_URL=https://ropsten.infura.io/v3/abc123abc123abc123abc123abc123ab
 SEPOLIA_URL=https://rpc.sepolia.dev
 ETHEREUM_URL=https://mainnet.infura.io/v3/abc123abc123abc123abc123abc123ab
 
-# Application config
+# Smart Contract Deployment config
+## MiniWallet
+MINIWALLET_INITIAL_OPERATOR_THRESHOLD=10
+MINIWALLET_INITIAL_OPERATORS=["0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266","0x70997970C51812dc3A010C7d01b50e0d17dc79C8","0x3C44CdDdB6a900fa2b585dd299e03d12FA4293BC","0x90F79bf6EB2c4f870365E785982E1f101E93b906"]
+MINIWALLET_INITIAL_USER_LIMIT=1000000
+MINIWALLET_INITIAL_AUTH_LIMIT=100000
 
-INITIAL_OPERATOR_THRESHOLD=10
-INITIAL_OPERATORS=["0x70997970C51812dc3A010C7d01b50e0d17dc79C8","0x3C44CdDdB6a900fa2b585dd299e03d12FA4293BC","0x90F79bf6EB2c4f870365E785982E1f101E93b906"]
-INITIAL_USER_LIMIT=1000000
-INITIAL_AUTH_LIMIT=100000
-TEST_INITIAL_OPERATOR_THRESHOLD=10
-TEST_INITIAL_OPERATORS=["0x70997970C51812dc3A010C7d01b50e0d17dc79C8","0x3C44CdDdB6a900fa2b585dd299e03d12FA4293BC","0x90F79bf6EB2c4f870365E785982E1f101E93b906"]
-TEST_INITIAL_USER_LIMIT=1000
-TEST_INITIAL_AUTH_LIMIT=100
\ No newline at end of file
+## Mini721
```

Comment:
There are too many components in this file. Can you break it down to multiple files? dotenv can load any number of environment files of any names

Reply:
Example code for multiple .env files (note to myself)
```
const dotenv = require('dotenv')
dotenv.config({ path: './.env' })
dotenv.config({ path: './t0.env' })
dotenv.config({ path: './t1.env' })
```